.github/workflows/build.yml

```diff
@@ -30,4 +30,9 @@ jobs:
       - uses: actions/checkout@v3
       - run: make
       - run: make test
+  integration:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - run: make
```

Comment:
Instead of adding `make`, maybe we can add the `pre-build` target to `make integration` in the Makefile. That'll just create the C library, and won't spend time building the binaries.

Reply:
Thanks will do.